tests/untestable/tssl_server.nim

```diff
@@ -0,0 +1,119 @@
+discard """
+  cmd: "nim $target --threads:on -d:ssl $options $file"
+"""
+
+#            Nim - Basic SSL integration tests
+#        (c) Copyright 2020 Nim contributors
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+## Runs a simple HTTPS server
+## Used by .github/workflows/ci_ssl.yml
```

Comment:
```suggestion
## Used by .github/workflows/ci_ssl.yml
```
doesn't exist anymore (and users could grep for the name to find where it's tested)